shared/common-adapters/index.stories.js

```diff
@@ -71,6 +72,7 @@ const load = () => {
     saveIndicator,
     sectionList,
     standardScreen,
+    switchStories,
```

Comment:
just `switch` for consistency

Reply:
unfortunately `switch` is a reserved keyword 😅